Documentation/packaging-guide.md

```diff
@@ -1,40 +0,0 @@
-# Packaging {#packaging-guide}
```

Comment:
Did this get moved somewhere?

Reply:
Ah I just looked at the issue this is based on and see it was removed intentionally -- ignore this!